src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -493,6 +495,45 @@ public void staticUserExport(OutputStream output, boolean convertHeader) throws
         }
     }
 
+    void userTeamsExport(OutputStream output, boolean convertHeader) throws IOException{
+        log.info("Exporting USER_TEAMS_DATA to CSV", value(EVENT, CSV_EXPORT_START));
+        OutputStream buffer = new BufferedOutputStream(output);
+        OutputStreamWriter outputWriter = new OutputStreamWriter(buffer, "UTF-8");
+        List<String> headers = Stream.of("User UUID", "Teams UUID").collect(Collectors.toList());
+
+        List<String> substitutedHeaders = headers;
+        if (convertHeader) {
+            substitutedHeaders = headerConverter.substitute(headers);
+        }
+        Set<UserWithTeamsDto> users = infoClient.getUsersWithTeams();
+
+        try (CSVPrinter printer = new CSVPrinter(outputWriter, CSVFormat.DEFAULT.withHeader(substitutedHeaders.toArray(new String[headers.size()])))) {
+            users.forEach((user) -> {
+                try {
+                    List<UUID> userTeams = user.getTeamUUIDs();
+                    final AtomicReference<String> teamList = new AtomicReference<>();
+                    teamList.compareAndSet(teamList.get(), "");
+                    userTeams.forEach((team) -> {
```

Comment:
Is this appending effectively doing `{team1Uuid}, {team2Uuid}`...

I'd have probably said having a new row per team such as:
{user1}, {team1}
{user1}, {team2}

Happy if the current solution is agreed - as it works, but we should pick this up for future...

Reply:
I'll make a note of this to review soon, not blocking for now.